motion.py

```
@@ -85,28 +85,31 @@ def isotropic_upsample_and_pad(image, interpolation=sitk.sitkBSpline5):
-        image.GetPixelID(),
-        useNearestNeighborExtrapolator=True,
-    )
-    resampled_image[resampled_image < 0] = 0
+    if np.allclose(original_spacing,image.GetDimension()*(min_spacing,)):
+        # the image is already isotropic
+        resampled_image = image
+    else:
+        # Compute new size to maintain physical extent
+        original_size = image.GetSize()
+        new_size = [
+            int(round(original_size[i] * original_spacing[i] / min_spacing))
```

Ruff (0.14.13)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)